tests/ecc_test.c

```diff
@@ -37,6 +37,93 @@ static unsigned int sizes[] = {
 #endif
 };
 
+const char* names[] = {
```

Comment:
`static` and a more meaningful name please

Reply:
That's what it was called when it was function-local ;-)
I'll rename it to `curvenames` and make it static.